app/pktgen-port-cfg.c

```diff
@@ -165,7 +165,7 @@ pktgen_config_ports(void)
             pinfo->seq_pkt = rte_zmalloc_socket(NULL, (sizeof(pkt_seq_t) * NUM_TOTAL_PKTS),
                                                 RTE_CACHE_LINE_SIZE, sid);
             if (pinfo->seq_pkt == NULL)
-                pktgen_log_panic("Unable to allocate %'ld pkt_seq_t headers", NUM_TOTAL_PKTS);
+                pktgen_log_panic("Unable to allocate %d pkt_seq_t headers", NUM_TOTAL_PKTS);
```

Comment:
On ubuntu 24.04 LTS x86_64 machine this change will cause an error to happen because NUM_TOTAL_PKTS appears to be (long int) value on this machine and build.

If we cast `(long int)NUM_TOTAL_PKTS` on the above line it builds on my machine still, does it build on your machine?